recipes/scapp/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "scapp" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: scapp
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/Shamir-Lab/SCAPP/archive/v0.1.2.tar.gz
+  sha256: ceeefdd15cfb6435bcd0092f56bc570d37eab469c8233f7c65e0c876ef3eaf5f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+  entry_points:
+    - scapp = scapp.scapp:main
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - blast
+    - bwa
+    - samtools
+    - plasclass
+    - networkx 2.4
+    - python >=3.7
+    - pysam 0.15.3
```

Comment:
Does it really need these exact versions of pysam and networkx, or should it be assumed that newer releases would also work?

Reply:
Probably okay to assume newer releases will also work.